control-plane-operator/controllers/hostedcontrolplane/oapi/oauth_deployment.go

```diff
@@ -74,6 +74,27 @@ func ReconcileOAuthAPIServerDeployment(deployment *appsv1.Deployment, ownerRef c
 		AutomountServiceAccountToken: pointer.BoolPtr(false),
 		Containers: []corev1.Container{
 			util.BuildContainer(oauthContainerMain(), buildOAuthContainerMain(p)),
+			{
+				Name:            "audit-logs",
+				Image:           p.Image,
+				ImagePullPolicy: corev1.PullIfNotPresent,
+				Command: []string{
+					"/usr/bin/tail",
+					"-c+1",
+					"-F",
+					"/var/log/openshift-oauth-apiserver/audit.log",
+				},
+				Resources: corev1.ResourceRequirements{
+					Requests: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("5m"),
+						corev1.ResourceMemory: resource.MustParse("10Mi"),
+					},
+				},
+				VolumeMounts: []corev1.VolumeMount{{
+					Name:      oauthVolumeWorkLogs().Name,
+					MountPath: "/var/log/openshift-oauth-apiserver",
```

Comment:
pls use util.PodVolumeMounts as with the main container. The reason to do that is that if in the future we change the path, we don't have to look multiple places to fix it.